Makefile

```
@@ -0,0 +1,24 @@
+#
+# Copyright 2022 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PROTOC_IMAGE := namely/protoc-all:sha256:07f1ba9dbe11f5675e2efc8617c9552217dc4c3eb5ccd108f7c3889878dbae50 # v1.50_0
+
+# generate Go protobuf code
+proto: 
+	@echo "Generating Protobuf files"
+	docker run -v ${PWD}:/defs ${PROTOC_IMAGE} -i protos -f envelope.proto -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
+	docker run -v ${PWD}:/defs ${PROTOC_IMAGE} -i protos -f sigstore_common.proto -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
+	docker run -v ${PWD}:/defs ${PROTOC_IMAGE} -i protos -f sigstore_rekor.proto -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
+	docker run -v ${PWD}:/defs ${PROTOC_IMAGE} -i protos -f sigstore_bundle.proto -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
```

Controversial change needed for this to work on Apple Silicon (feel free to ignore if this breaks other environments):
```suggestion
	docker run --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -i protos -f envelope.proto -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
	docker run --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -i protos -f sigstore_common.proto -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
	docker run --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -i protos -f sigstore_rekor.proto -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
	docker run --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -i protos -f sigstore_bundle.proto -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
```

---

BTW, it does look like you can pass multiple `-f <file>` argument-pairs to protoc, so this can all be one docker invocation:

```
docker run --platform linux/amd64 -v ${PWD}:/defs ${PROTOC_IMAGE} -i protos -f sigstore_common.proto -f envelope.proto -f sigstore_rekor.proto -f sigstore_bundle.proto -l go --go-module-prefix github.com/sigstore/protobuf-specs/gen/pb-go
```

---

Oh yeah! I can't remember why I didn't do that. I think I just wrote the syntax wrong and thought it wouldn't work.
done
the platform doesn't break for me so added